bindinfo/handle.go

```diff
@@ -60,20 +67,28 @@ type BindHandle struct {
 		atomic.Value
 	}
 
-	parser         *parser.Parser
-	lastUpdateTime types.Time
+	Parser *parser.Parser
 }
 
-// NewBindHandle creates a new BindHandle.
-func NewBindHandle(ctx sessionctx.Context, parser *parser.Parser) *BindHandle {
-	handle := &BindHandle{parser: parser}
+// NewBindHandle creates a new GlobalBindHandle.
+func NewBindHandle(parser *parser.Parser) *BindHandle {
+	handle := &BindHandle{}
+	handle.Parser = parser
+	handle.bindInfo.Value.Store(make(cache, 32))
```

Comment:
What about only allocate it on demand for session handle?

Reply:
OK